src/Microsoft.Extensions.FileProviders.Abstractions/NullFileProvider/NullFileInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Though currently Sources depend on Abstractions so there would be a circular dependency there. I think we should just move the NullFileProvider to Sources as well since it is an implementation anyways.